ppb/sprites.py

```diff
@@ -139,17 +182,27 @@ class RectangleShapeMixin:
 
     Classes derived from RectangleShapeMixin default to the same size and
     shape as all ppb Sprites: A 1 game unit by 1 game unit square. Just set
-    the width and height in your constructor (Or as class attributes) to
-    change this default.
+    the width and height in your constructor (Or as
+    :class:`class attributes <BaseSprite>`) to change this default.
+
+    .. warning:: The concrete class using RectangleShapeMixin must have a
```

Comment:
A warning seems a little bit dire for this? I also feel like we should note that this is provided by the default stuff.